dataactbroker/app.py

```diff
@@ -103,7 +83,7 @@ def runApp():
 
 if __name__ == '__main__':
     runApp()
-    proc = multiprocessing.Process(target=checkDynamo)
+    proc = multiprocessing.Process()
```

Comment:
@jworcestBAH This was code that Nipun added specifically for local broker instances. It uses a separate process to do a Dynamo call...if nothing comes back in 5 seconds (or whatever the time is), the app returns reminder to start dynamo. We added this code b/c the local app would just hang with no messages if Dynamo wasn't started.